docs/technical_specs/model_libraries/gas_solid_contactors/unit_models/fixed_bed_0D.rst

```diff
@@ -0,0 +1,168 @@
+  This is provided as a Physical Parameter Block by the Flowsheet when creating the model. 
+  If a value is not provided, the ControlVolume Block will try to use the default property package if one is defined.
+* solid_property_package_args - set of arguments to be passed to the solid phase Property Blocks when they are created (default = 'None').
+* reaction_package - reaction package to use when calculating properties for solid phase reaction (default = None). 
+  This is provided as a Reaction Parameter Block by the Flowsheet when creating the model. 
+  If a value is not provided, the ControlVolume Block will try to use the default property package if one is defined.
+* reaction_package_args - set of arguments to be passed to the Reaction Blocks when they are created (default = None).
+
+Constraints
+-----------
+
+In the following, the subscripts :math:`g` and :math:`s` refer to the gas and solid phases, respectively. 
+FixedBed0D units write the following Constraints:
+
+Reaction and Mass/Heat Transfer Constraints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Solid material holdup:
+
+.. math:: h_{mass,s,t,j} = V_{s} \rho_{mass,s,t} x_{mass,s,t,j}
+
+Solid material accumulation:
+
+.. math:: {\dot{h}}_{mass,s,t,j} = V_{s} {MW}_{s,j} {\Sigma}_{r} {r_{s,t,r} \nu_{s,j,r}}
+
+Total mass of solids
+
+.. math:: S_{s,t} = V_{s} {\Sigma_{j} {\rho_{mass,s,t,j}}}
+
+Sum of component mass fractions
+
+.. math:: {\Sigma_{j}} {x_{mass,s,t,j}} = 1 \space \space \space \forall \space t
+
+Solid energy holdup:
+
+.. math:: h_{energy,s,t} = V_{s} \rho_{mass,s,t} H_{mass,s,t}
+
+Solid energy accumulation:
+
+.. math:: {\dot{h}}_{energy,s,t} = - V_{s} {\Sigma}_{r} {r_{s,t,r} H_{rxn,s,t,r}}
+
+List of Variables
+-----------------
+
+.. csv-table::
+   :header: "Variable", "Description", "Reference to"
+
+   ":math:`H_{mass,s,t}`", "Solid phase mass enthalpy", "``solids.enth_mass``"
+   ":math:`H_{rxn,s,t,r}`", "Solid phase reaction enthalpy", "``solids.reactions.dh_rxn``"
+   ":math:`h_{energy,s,t}`", "Energy holdup, solid phase", "``solids.energy_material_holdup``"
+   ":math:`h_{mass,s,t,j}`", "Material holdup, solid phase", "``solids.solids_material_holdup``"
+   ":math:`{\dot{h}}_{energy,s,t}`", "Energy accumulation, solid phase", "``solids.solids_energy_accumulation``"
+   ":math:`{\dot{h}}_{mass,s,t,j}`", "Material accumulation, solid phase", "``solids.solids_material_accumulation``"
+   ":math:`r_{s,t,r}`", "Solid phase reaction rate of reaction r", "``solids.reactions.reaction_rate``"
+   ":math:`S_{s,t}`", "Total mass of solids", "``solids.mass_solids``"
+   ":math:`T_{s,t,x}`", "Solid phase temperature", "``solids.temperature``"
+   ":math:`x_{mass,s,t,j}`", "Mass fraction of component j, solid phase", "``solids.mass_frac_comp``"
+   ":math:`V_{s}`", "Total volume of solids", "``solids.volume_solid``"
+   "*Greek letters*", " ", " "
+   ":math:`\varepsilon`", "Reactor bed porosity", "``particle_porosity``"
```

Comment:
`varepsilon` is the particle porosity. The reactor bed porosity is not used in this model